plugins/zoom/icon_zoom/util/api.py

```diff
@@ -167,22 +225,29 @@ def _handle_response(self, response: Response, allow_404: bool, original_call_ar
             return response.json()
 
     @staticmethod
-    def _handle_rate_limit_error(response: Response):
+    def _handle_rate_limit_error(response: Response) -> PluginException:
         rate_limit_type = response.headers.get("X-RateLimit-Type", "")
         rate_limit_limit = response.headers.get("X-RateLimit-Limit")
         rate_limit_remaining = response.headers.get("X-RateLimit-Remaining")
         rate_limit_retry_after = response.headers.get("Retry-After")
 
         if rate_limit_type == "Light":
-            raise PluginException(
+            return PluginException(
                 cause="Account is rate-limited by the maximum per-second limit for this API.",
```

Comment:
Just wondering if we hit the max per-second limit for the API should we wait and retry after the specified interval?

Reply:
I talked with @dgreene-r7 about this and he wants us to just raise an exception.